src/main/resources/db/migration/V0004__Create_table_animals.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE `animals` (
```

Comment:
Why we divided sql queries to more one file ?

Reply:
I guess that for migration and develop DB must be more than  one file.Step by stap. If I am wrong, I will make one file.

Reply:
done